### DISCUSSION TITLE
502 bad gateway after update

### DISCUSSION THREAD

**Discussion by edigdog** [NightscoutFoundation/xDrip#3242] [General]
Hi All, need urgent help.
After updating from 14.2.6 to whatever the latest is using google cloud I keep getting bad gateway even though my status page has no errors etc.
I wish I didn't update now, everything was working perfectly. Please help
I have done all the steps in update process 
![IMG_3554](https://github.com/NightscoutFoundation/xDrip/assets/141674767/0d752aee-55f7-4fe7-850b-6e65d1cb4ad0)

**Comment by Navid200** [comment]
Can you show the logs?  It's item 2 on the main menu.

**Reply by edigdog** [reply]
<img width="856" alt="Screenshot 2023-12-16 at 16 53 00" src="https://github.com/NightscoutFoundation/xDrip/assets/141674767/37592aa0-06c2-4c8b-a006-61a5909a74fe">

**Reply by Navid200** [reply]
Please run phase 1 from the Google Cloud Setup submenu.

After phase 1 completes and you get back to the main menu, restart the server.
Then, wait a few minutes and your Nightscout should come up.

**Reply by edigdog** [reply]
Okay thanks, wasn't sure if that would break everything, will try now

**Reply by edigdog** [reply]
Jesus, thanks mate. All working again.......
The instructions for update don't mention running phase 1 again, so wasn't sure I should

**Reply by Navid200** [reply]
It's not easy.

I need to add a detector to identify this situation and put a marker on the status page.

I'll see what I can do to improve this.
Thanks

**Reply by edigdog** [reply]
I can't thank you enough, panicked as this screwed all my sons 'management' up.
I will close discussion.
